lollipop/types.py

```diff
@@ -1292,3 +1293,42 @@ def __repr__(self):
             klass=self.__class__.__name__,
             inner_type=repr(self.inner_type),
         )
+
+
+class Transform(Type):
```

Comment:
Commit-wise, this should be a separate commit named something like "Add Transform modifier type", not a part of commit titled "minor errors in types.py fixed"